generators/php/codegen/src/ast/Trait.ts

```diff
@@ -0,0 +1,101 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+import { Field } from "./Field";
+import { Method } from "./Method";
+import { Comment } from "./Comment";
+import { orderByAccess } from "./utils/orderByAccess";
+import { ClassReference } from "./ClassReference";
+
+export declare namespace Trait {
+    interface Args {
+        /* The name of the PHP trait */
+        name: string;
+        /* The namespace of the PHP trait */
+        namespace: string;
+        /* Docs associated with the trait */
+        docs?: string;
+        /* The traits that this trait uses, if any */
+        usedTraits?: ClassReference[];
```

Comment:
Same thing with `usedTraits` -> `traits` here.

Reply:
yeah good call, i wasn't sure if it was weird for a `Trait` field to have a `traits` field, but with your blessing i'm comfortable with it :)